Assets/MixedRealityToolkit.Services/DiagnosticsSystem/MixedRealityToolkitVisualProfiler.cs

```diff
@@ -457,7 +461,7 @@ private static ulong AppMemoryUsageLimit
             get
             {
 #if WINDOWS_UWP
-                return Windows.System.MemoryManager.AppMemoryUsageLimit;
+                return MemoryManager.AppMemoryUsageLimit;
```

Comment:
>MemoryManager [](start = 23, length = 13)

nit: inline namespace usage for WINDOWS_UWP feels less error prone. If anyone creates a MemoryManager class collisions will only be observed when compiling compared to in the editor. From what i've experienced, most folks don't necessarily compile WINDOWS_UWP when attempting to submit changes, so it may be better to keep the namespace declaration in line as a safeguard for type collisions.

Reply:
Inline doesn't work in this case because the namespace for this class contains Windows; I can do a using here again with WMemoryManager

---
In reply to: [267121827](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3658#discussion_r267121827) [](ancestors = 267121827)

Reply:
I understand that this would still need to be behind the WINDOWS_UWP macro. I just think that having the full namespace declared here behind the macro might avoid some collision breaks if someone makes a memorymanager class. that going said, this if people consistently compile for visual studio like they should to test changes, its not necessarily an issue. And my preference here is not strong enough to block on checking in. :P

---
In reply to: [267123037](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3658#discussion_r267123037) [](ancestors = 267123037,267121827)

Reply:
I can update the MRTK.Windows to MRTK.UWP and this issue I described above would no longer happen

---
In reply to: [267123833](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3658#discussion_r267123833) [](ancestors = 267123833,267123037,267121827)

Reply:
synced offline and now understand that the main concern is that Windows exists in MRTK world. This is fine as is.

---
In reply to: [267123833](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3658#discussion_r267123833) [](ancestors = 267123833,267123037,267121827)